plugins/sinks/wavefront/wavefront.go

```diff
@@ -110,11 +110,24 @@ func (sink *wavefrontSink) send(batch *metrics.DataBatch) {
 
 	before := errPoints.Count()
 	for _, point := range batch.MetricPoints {
-		if point.Tags == nil {
-			point.Tags = make(map[string]string, 1)
+		tags := make(map[string]string)
```

Comment:
move this out of the for loop and reuse?

Reply:
1. Reusing the map implies a map cleanup, and the will be way more expensive than create a new on.
2. If we change the SDK logic on the future (ie, to keep the map for some time) can cause mixing metrics tags .